### ISSUE TITLE
Assign closed bugs, feature requests, and PRs to current milestone

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode-github-triage-actions#24]
I definitely mess this up a couple of times and have even had some bugs slip past verification as I forgot to assign them to the milestone. It would be nice if the bot could assign things like bugs, feature requests, and PRs to the milestone once they've been completed. We wouldn't want this for all labels such as `duplicate`, `needs more info`, or failed `backlog-candidates`.

**Comment by JacksonKearl** [comment]
We could probably assign things that were closed with a commit to the milestone, though we don't actually have a concept of "current milestone" so that might take some doing. Would that work for your cases?

**Comment by lramos15** [comment]
Yeah that works. My guess is that you could probably calculate when the milestone via the plan. I'm also curious if this would annoy people but in theory it wouldn't trigger if a milestone was already assigned.
